F2Py.cmake

```diff
@@ -89,7 +89,7 @@ function (f2py_add_module target_name)
     # Definitions for source files processing
     set(fortran_defs)
     foreach(_def ${F2PY_ADD_MODULE_COMPILE_DEFS})
-      STRING(APPEND fortran_defs "-D${_def} ")
+      list(APPEND fortran_defs "-D${_def}")
```

Comment:
Is "list" expanded in `add_custom_command` below to just string of items separated by spaces? Because usually it is represented as a string separated by semicolon.

Reply:
Options for commands need to be passed as lists. There won't be semicolons. The string append was wrong, it generated this `-DIMPY\ -DFOO` instead of the desired `-DIMPY -DFOO`. I am surprised that no one noticed this error before.